lib/api/messages.js

```diff
@@ -1614,6 +1619,8 @@ module.exports = (db, server, messageHandler, userHandler, storageHandler, setti
                 });
             }
 
+            let attachmentCharset = getAttachmentCharset(messageData.mimeTree, attachment)[0];
```

Comment:
If the response is always an array, then using a destructuring assignment would be more readable

```
const [attachmentCharset] = getAttachmentCharset(messageData.mimeTree, attachment)
```